### ISSUE TITLE
Code analysis breaks when using parentheses.

### ISSUE THREAD

**Issue by Jezza** [movatechtoday/jsonnet_templating_language_support#4]
A simple snippet like:

```jsonnet
local auth = (import 'libs/auth.libsonnet').new(ctx);
local thing = auth.field; // <-- It has no idea what field is here.
```

But if we expand it to:
```jsonnet
local auth_lib = import 'libs/auth.libsonnet';
local auth = auth_lib.new(ctx);
local thing = auth.field; // <-- It can now resolve it.
```

**Comment by movatechtoday** [comment]
Hello @Jezza 
Sorry for late response. The issue has been fixed in the latest release -- 2022.1.9. 

Also, it has a lot of additional features.

### Jsonnet core
- Completion/navigation/Find usages for field names inside `super` statement (nested elements is not supported at the moment)
- An inspection that checks arguments usages
  - Verifies if the specified number of arguments is less/more than the number of parameters
- Improved Code Analysis engine. Now it understands parenthesized expressions.
- Added error reporter. If you encounter an error inside the plugin, the IDE will prompt you to send a report to MovaTech
- Added missing `std.sum` method to the standard library.
- Now documentation is shown for items inside the completion popup.

### Grafana Tanka
- Add completion for native function names inside calls `std.native()`

### Other
- Added Drone CI icons

**Comment by Jezza** [comment]
Thanks for the response!

I finally found time to update, and can confirm that it partially works now.
